.github/workflows/publish.yml

```diff
@@ -0,0 +1,108 @@
+---
+name: Publish
+run-name: Publish ${{ inputs.publish_type }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      publish_type:
+        description: "Publish Options"
+        default: "Initial Publish"
+        type: choice
+        options:
+          - Initial Publish
+          - Redeploy
+          - Dry Run
```

Comment:
❓  Will this template out to "Publish Initial Publish", "Publish Redeploy", or even "Publish Dry Run" and is that intended?

Reply:
Yes, it's similar to what we do for [Server](https://github.com/bitwarden/server/actions/workflows/release.yml), but `Publish` instead of `Release`.  I had an issue with this in the `clients` repository, but it was because the input value was passed to our `release-version-check` action.  Since it isn't being passed here, this should be fine.  I agree the wording could be better, but I don't have any suggestions off the top of my head.